### ISSUE TITLE
Poly diff to list changed bases/components for CI testing

### ISSUE THREAD

**Issue by BeRT2me** [DavidVujic/python-polylith#116]
**Is your feature request related to a problem? Please describe.**
`poetry poly diff -s` is an amazing tool for a CI environment, but when working in a large project where tests for components/bases can become rather extensive, it would be nice to be able to get a simple list of those which were updated as well. (Rather than having to run _all_ tests)

**Describe the solution you'd like**
Something like:
`poetry poly diff bases` --> `base1 base3`
`poetry poly diff components` --> `component2 component4`

**Describe alternatives you've considered**
Maybe `-s` flag, and/or `bases`/`components` as flags rather than an argument.

Nice to have would be a list of test paths, so you could just run :
```
poetry run pytest `poetry poly diff --test-paths`
```

**Additional context**
```
# Test projects by using their self-defined dependencies.
for val in `poetry poly diff -s`
do
    poetry install --sync -C ./projects/${val}
    poetry install --only=dev
    for base in `poetry poly diff bases -C ./projects/${val}`
    do
        poetry run pytest ./test/proj/bases/${base}
    done
done

cd test/proj/components && poetry run pytest `poetry poly diff components`
```

**Comment by DavidVujic** [comment]
Good ideas @BeRT2me!

I have thought about this, and ways of implementing a `poly test` command that should be able to run tests for the changed bricks only. Inspired by the Clojure tooling: https://polylith.gitbook.io/poly/commands#test

This will probably be doable when using the Polylith structure also for the tests,
such as `test/<component-or-base>/<top-namespace>/<brick-name>`. 

If not disabled in the `workspace.toml`, a unit test template file is created when running `poetry poly create` for a component or base. Another idea I have is to add Pytest markers to the template (but then the tool will assume that the test tool is actually pytest).

**Comment by DavidVujic** [comment]
Hello again @BeRT2me!

I have added a new option to the `poly diff` command, to make it easier running tests for changed code only. I hope this will be useful for you. It is available in v1.11.0 of the tool.

Here's updated docs about the option and how it can be used for testing with `pytest`:
https://davidvujic.github.io/python-polylith-docs/commands/#testing
